internal/coreinternal/processor/filterconfig/config.go

```diff
@@ -90,6 +90,7 @@ type MatchProperties struct {
 	// This is an optional field.
 	SpanNames []string `mapstructure:"span_names"`
 
+	SpanDuration *DurationProperties `mapstructure:"span_duration"`
```

Comment:
This is self-explanatory but for convenience I think we should include the comment above